### PR TITLE
feat(frontend): Add support for failure modules

### PR DESCRIPTION
**Pull request by fatonramadani**
Screenshot 2022-09-26 at 14 11 31
Screenshot 2022-09-26 at 14 11 43

Screenshot 2022-09-26 at 14 11 59

### CODE REVIEW

**Comment by rubenfiszel** [comment]
I think it's missing FlowViewer and FlowStatusViewer required changes

File: frontend/src/lib/components/FlowModulesViewer.svelte [hunk 1]
```diff
@@ -98,4 +102,84 @@
 			</div>
 		</li>
 	{/each}
+	{#if failureModule}
+		<li class="w-full">
```
Comment (rubenfiszel) [inline frontend/src/lib/components/FlowModulesViewer.svelte:106]: are there no common parts that we can factorize here in a separate component ?

**Review by rubenfiszel — APPROVED** [review]
Looks great, only question is about the potential to refactor some duplicated code

File: frontend/src/lib/components/FlowModulesViewer.svelte (hunk 1 above)
Comment (fatonramadani) [reply frontend/src/lib/components/FlowModulesViewer.svelte:106]: Sure, I'll extract the common code in a sub component.

**Pull request merged**